tools-kmod/pkg.yaml

```diff
@@ -12,22 +22,13 @@ steps:
     prepare:
       - |
         tar -xJf kmod.tar.xz --strip-components=1
-
-        patch -p1 < /pkg/patches/strndupa.patch
-
-        mkdir build
-        cd build
-        ../configure \
-            --prefix=/usr \
-            --disable-manpages
+        meson setup build --prefix=/usr -Dmanpages=false
     build:
       - |
-        cd build
-        make -j $(nproc)
+        meson compile -C build
```

Comment:
let's also add `-j $(nproc)`

Reply:
it uses ninja, which parallelizes by default IIRC